common/changes/@microsoft/api-documenter/ts-4.4_2021-09-19-10-42.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/api-documenter",
+      "comment": "Update to TypeScript 4.4",
```

Comment:
Is this changelog message useful to consumers of these packages?

Reply:
Not really, but the CI was complaining that the changelog wasn't generated.

Reply:
You can generate blank changelog messages with `rush change --bulk --bump-type none`

Reply:
Done.